.eslintrc.json

```diff
@@ -44,6 +44,16 @@
         ],
         "enforceBuildableLibDependency": true
       }
-    ]
+    ],
+    "@typescript-eslint/ban-types": [
+      "error",
+      {
+        "types": {
+          "null": false
+        },
+        "extendDefaults": true
+      }
+    ],
+    "capitalized-comments": "off"
```

Comment:
Your change is only going to take effect in core-utils. Assuming you want to pull this change into other services, you need to make the change at `packages/eslint-config/src/index.js`.